index.bs

```diff
@@ -1821,16 +1821,21 @@ Charter Refinement</h3>
 	The [=charter refinement=] phase concludes
 	when there is a [=Team Decision=]
 	(informed by the work of the [=Chartering Facilitator=])
-	either to initiate [=AC Review=] of the [=charter draft=],
-	or to abandon the proposal.
+	to either complete charter refinement by initiating [=AC Review=] of the [=charter draft=],
+	or abandon the proposal.
 
 	[=Formal Objections=] filed during the [=charter refinement=] phase
 	are specially handled:
-	* Objections to abandoning the proposal can be appealed only if 5 or more Members,
+	* Objections to abandoning the proposal or to extending the refinment period can be appealed only if 5 or more Members,
 		through their [=Advisory Committee representative=],
-		formally object to the decision within 8 weeks of the decision being announced.
-		In this case, the [=Team=] must either resume charter refinement
-		or refer the matter to the [=Council=].
+		formally object to the decision within 8 weeks of the decision.
+		In this case,
+		the [=Team=] <em class=rfc2119>must</em> either
+		abide by their request
+		if all objections agree on the alternative course of action
+		(i.e. whether to abandon, extend, or complete charter refinement),
+		initiate [=AC Review=],
+		or convene a [=Council=] which then decides the course of action.
```

Comment:
Are the AC Review or Council available to the Team if all objectors agree? Or *must* the Team abide by such unanimous request? Some further rewording and/or markup may be appropriate, either way, but I think we're getting closer to communicating your intended meaning. I hope my `.bs` is correct.
```suggestion
		the [=Team=] <em class=rfc2119>must</em> do one of the following:
		* If all objectors agree on the alternative course of action
			(i.e., whether to abandon, extend, or complete charter 
			refinement), abide by their request.
		* Initiate an [=AC Review=].
		* Convene a [=Council=].
		If an [=AC Review=] is initiated or a [=Council=] is convened, that
		group then decides the subsequent course of action.
```

Reply:
Basically, there's three routes:
- 1) The objectors have convinced the Team that their preferred option is best. The Team enacts it.
- 2) The objectors have not convinced the Team, or have divergent views, and the Team has determined the best way forward is AC Review to formally solicit the input of the community and take a W3C Decision. The Team starts it.
- 3) The objectors have not convinced the Team, or have divergent views, and the Team also doesn't think AC Review is the best next step, so asks the Council (a "higher authority") to make a decision.

Reply:
I believe those three routes are found in [the current suggestion](https://github.com/w3c/process/pull/1012#discussion_r2023714057). Still, maybe this revision gets that message across better?
```suggestion
		the [=Team=] <em class=rfc2119>must</em> do one of the following:
		* Abide by the objectors' request, if they all agree on the
			alternative course of action (e.g., to abandon,
			extend, or complete charter refinement).
		* Initiate an [=AC Review=] to formally solicit the input of
			the community and take a W3C Decision on the subsequent
			course of action.
		* Convene a [=Council=] to decide the subsequent course of action.
```